### PR TITLE
Setup project & dependencies added: emotion/core - jest- reach/router

### PR DESCRIPTION
**Pull request by dotconde**
# Content:
* Setup for create-react-app project
* Dependencies added: emotion-core, jest & reach-router
* Git flow (master -> development -> features)

### CODE REVIEW

File: simcase-game/public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (test0n3) [inline simcase-game/public/manifest.json:1]: We have not defined PWA characteristics yet. We have to make some time to add characteristics, even prepare a proper manifest.

**Review by CesarCachay — CHANGES_REQUESTED** [review]
What about adding the others libraries we will need like react-router, jest & emotion/core?

File: simcase-game/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "simcase-game",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.8.6",
```
Comment (test0n3) [inline simcase-game/package.json:10]: We need to previously declare all the requirements me need.
Don't forget there's a difference between _dependencies_ and _devDependencies_

**Review by test0n3 — DISMISSED** [review]
All is fine, just we have to keep in mind the Git flow and avoid at all costs doing PR at the last minute

**Review by CesarCachay — APPROVED** [review]

**Review by test0n3 — APPROVED** [review]

File: simcase-game/public/manifest.json (hunk 1 above)
Comment (dotconde) [reply simcase-game/public/manifest.json:1]: Definitely yes, we will check that feature in week 3.

**Pull request merged**